### DISCUSSION TITLE
Prompt for Mapping, Localization and Navigation in Field Robotics

### DISCUSSION THREAD

**Discussion by Prof-pengyin** [microsoft/PromptCraft-Robotics#4] [LLM + Field robots]
Hi Guys,

We would like to create a prompt for our several GitHub Repos under MetaSLAM:

1. [AutoMerge_Server](https://github.com/MetaSLAM/AutoMerge_Server), which is a crowdsourced mapping package for multi-agent mapping and localization.
2. [iSimLoc](https://github.com/MetaSLAM/iSimLocServer), which is a UAV localization system with Google Satelite Images without GPS.
3. [BioSLAM](https://github.com/MetaSLAM/BioSLAM_server), which is a lifelong learning system for incremental perception.
4. [MUI-TARE](https://github.com/MetaSLAM/MUI-TARE_Server), which is a multi-agent exploration approach.

Our vision is to enhance the Field of Robotics research with the powerful ChatGPT ability for the Robotics Community.
For the more detail, please refer to our website:

- [MetaSLAM Github](https://github.com/MetaSLAM)
- [MetaSLAM website](https://metaslam.github.io/)

Max Yin
Carnegie Mellon University, RI.

**Comment by saihv** [comment]
Hi, thanks for the interest! Could you clarify a bit more on how you'd like to see ChatGPT potentially integrate with a SLAM pipeline? Do you mean something like using a ChatGPT interface to initialize and run a SLAM pipeline and then for example, explore a space? 

Please note that the prompts we discuss in the paper/this repo are more geared towards perception-action loops, where ChatGPT understands human instructions and executes robot action APIs. Would be great if you can briefly explain an example scenario on how ChatGPT helps with a SLAM/localization pipeline.
